Tasks/Xcode/xcode.ts

```diff
@@ -213,7 +213,20 @@ async function run() {
         }
 
         //--- Xcode Build ---
-        await xcb.exec();
+        let errText: string = '';
+        xcb.on('stderr', (data: Buffer) => {
+            errText += data.toString();
```

Comment:
I'm fine with this change.  I'd prefer the check to be here, though.  Save a flag that this was tripped or not.  Rather than load all the log into memory before checking.

Reply:
I think that allows you to not have to buffer.  I think that output will always be on a line.

Reply:
We also considered a general feature in task json of regex to guidance map and we do the on line handling